### PR TITLE
Josue login

### PR DESCRIPTION
**Pull request by josueaceves**
@CorainChicago  @dinkengraven

### CODE REVIEW

File: mikehub/spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ #    it "is valid when the user has a title" do
+ #        user = User.new
+ #        user.valid?
+ #        expect(user.errors).to_not be_empty
+ #    end
+
+ #  end
+
+ describe User do
+    it "should have many teams" do
```
Comment (CorainChicago) [inline mikehub/spec/models/user_spec.rb]: Users don't have teams - they have edits and articles.  I'm confused, please explain

File: mikehub/app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  def create
+
+    user = User.find_by(email: user_params[:email])
+    if user && user.authenticate(user_params[:password])
+
+      session[:user_id] = user.id
+      redirect_to '/users/new'
+    else
+
```
Comment (CorainChicago) [inline mikehub/app/controllers/sessions_controller.rb:17]: extra space

File: mikehub/app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  def login
+  end
+
+
+
+  def create
+
+    user = User.find_by(email: user_params[:email])
+    if user && user.authenticate(user_params[:password])
+
```
Comment (CorainChicago) [inline mikehub/app/controllers/sessions_controller.rb:13]: extra space

File: mikehub/config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
 
 
+  get 'sessions/login' => 'sessions#login'
+
+  post 'sessions/create' => 'sessions#create'
+
+  get 'sessions/logout' => 'sessions#logout'
+
   root "pages#index"
```
Comment (CorainChicago) [inline mikehub/config/routes.rb:11]: Nice routes

**Comment by CorainChicago** [comment]
I an confused on one rspec test.

**Comment by josueaceves** [comment]
@McEileen merge master please do your magic :D

**Pull request merged**